card/mod/04_settings/set/self/recaptcha_public_key.rb

```diff
@@ -0,0 +1,9 @@
+event :update_recaptcha_public_key, :after=>:store do
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.